server/routes/index.ts

```diff
@@ -32,6 +33,9 @@ export default function routes(services: Services): Router {
     res.redirect(config.learningPlanUrl)
   })
 
+  router.get('/plan/create/:id/**/update', [getCiagPlanByIdResolver(services.ciagService)])
```

Comment:
The last part of the fix is here. This uses wildcard paths to setup the routes, but only those with `update` as the last element. For those, given that we know the journey is an update journey, we use the resolver to retrieve the CIAG Plan (Induction)


(don't worry - the terminology `CIAG Plan` will soon be a thing of the past, to be replaced with `Induction` 😁 )